docs/plugin_developer_guide.md

```diff
@@ -313,87 +315,87 @@ func (demo *DemoPlugin) Run(context plugin.PluginContext, args []string){
+- Use abbreviation only when it’s commonly accepted, for example use 'vpn' for Virtual Private Network service.
 
-To name the commands:
+**To name the commands**:
 
 - Use lower case words and hyphen
 - Follow a “noun-verb” sequence
-- For command that list objects, use plural of the object name (or "object-list” if plural won’t work)
-- Use common verbs like add, create, bind, update, delete …
+- For commands that list objects, use the plural of the object name, such as `objects`. If a plural will not work, use the name and list, such as `object-list`.
+- Use common verbs such as add, create, bind, update, delete …
 
-For example, the following command names are correct:
+The following command names are formatted correctly, and provides a possible description for the command:
 
-route-map
+- `route-map`: Map out the route from one place to another in a location application.
 
-route-unmap
+- `route-unmap`: Unmap a previously mapped route.
 
-template-run
+- `template-run`: Run a selected template.
 
-plugin-repo-add
+- `plugin-repo-add`: Add a plug-in repository.
 
-plugin-repo-remove
+- `plugin-repo-remove`: Remove a plug-in repository.
 
-templates
+- `templates`: Get a list of templates.
 
-...
-
-The following command names are invalid:
+The following command names are invalid, and explain why:
 
-Map-Route
+- `Map-Route`: Uses uppercase letters (M, R). Does not follow "noun-verb" word order (the route needs to be mapped, but as this is written, the order implies that the map needs to be routed).
 
-map-route
+- `map-route`: Does not follow "noun-verb" word order (the route needs to be mapped, but as this is written, the order implies that the map needs to be routed).
 
-map route
+- `map route`: Uses a space instead of a hyphen. Does not follow "noun-verb" word order (the route needs to be mapped, but as this is written, the order implies that the map needs to be routed).
 
-route map
+- `route map`: Uses a space instead of a hyphen.
 
-route\_map
+- `route\_map`: Uses unallowed characters (`\_`) instead of a hyphen.
 
-add-plugin-repo
+- `add-plugin-repo`: Does not follow "noun-verb" word order. The verb "add" should be the last part of the command name.
 
-plugin-add-repo
+- `plugin-add-repo`: Does not follow the "noun-verb" word order. The verb "add" should be the last part of the command name.
 
-...
 
-Use single quotation marks (') around the command in output message and the command itself should be yellow with **bold**. For example:
+**Command formatting in output messages**:
+Use single quotation marks (') around the command name and options in output message. The command itself should be yellow with **bold**.  Where possible, place command names at the end of the sentence, not in the middle. For example:
 
 ```
-You have not logged in. Use 'bluemix login'to login first.
+You are not logged in. Log in by running 'bluemix login'.
```

Comment:
So this is one of the style changes that I made: Moving command names to the end of the sentence when possible. It's based off [IBM Style guidelines](https://apps.na.collabserv.com/wikis/home?lang=en-us#!/wiki/W580a84a67242_4da6_81cf_69e3db4bdbfa/page/Command%20syntax).